data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_templates.lst

```diff
@@ -1 +1,5 @@
 Dragonblood					VISIBLE:NO																																																																																					RACESUBTYPE:Dragonblood
+Favclass_Ranger		VISIBLE:NO	FAVOREDCLASS:Ranger
+Favclass_Sorcerer	VISIBLE:NO	FAVOREDCLASS:Sorcerer
```

Comment:
Why are we reinventing the wheel here?

ph_templates.lst has all the base class favored classes. Ranger and Sorcerer are:
Favored Class ~ Ranger
Favored Class ~ Sorcerer